From aa96bdb9c26ea78f2fbc271fa70f096134e3b85e Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Fri, 4 Oct 2019 21:00:05 -0400
Subject: [PATCH 5/7] Allow broken phony targets

This relaxes the constraints when running ckati and in particular allows
to generate output files outside of $OUT or using absolute paths.
---
 BoardConfig.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index fac4810..453b513 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -91,3 +91,4 @@ BOARD_SEPOLICY_DIRS += device/generic/common/sepolicy \
                        system/bt/vendor_libs/linux/sepolicy \
 
 BOARD_BUILD_SYSTEM_ROOT_IMAGE:= true
+BUILD_BROKEN_PHONY_TARGETS := true
-- 
2.21.0

